### PR TITLE
Removes `bold` from font-face declaration

### PR DESCRIPTION
**Pull request by TheSonOfThomp**
`bold` alias defaults to 700 in Chrome, Safari, FF & Edge

## ✍️ Proposed changes

🎟 _Jira ticket:_ [Name of ticket](https://jira.mongodb.org/browse/[name-of-ticket])

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Style change (purely visual updates; if this is the case, attach a screenshot below!)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

## Feedback Levels

Please try to mark your comments with one of these levels

1. Praise (👏 🚀 🔥 🍾)
2. Opinions, Ideas, Suggestions (💡 👀)
3. Let’s discuss @ Code review (🤷 🤔 🧐 )
4. Doesn’t meet agreed upon standards (😬 ⚠️ ‼️)
5. Missing an edge case (🛑 🙅 ❌)
6. Missing core functionality (⛔️ ☣️ 🚧 🌋 💀)

(Note: Use ⌘-⌃-␣ to open the Emoji keyboard!)

### CODE REVIEW

File: .storybook/manager-head.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
         format('woff2'),
       url('https://d2va9gm4j17fy9.cloudfront.net/fonts/euclid-circular/EuclidCircularA-Semibold.ttf')
         format('truetype');
-    font-weight: 700, bold;
+    font-weight: 700;
```
Comment (TheSonOfThomp) [inline .storybook/manager-head.html:17]: Removing bold since major browsers map `bold` to 700

**Review by spark33 — APPROVED** [review]

File: .storybook/manager-head.html (hunk 1 above)
Comment (spark33) [reply .storybook/manager-head.html:17]: Why is this change necessary? I'm approving the change, but I feel like we should all know why this is being done.

Should this change also be applied to the website? Looks like we've got `700, bold` defined there too.

Comment (TheSonOfThomp) [reply .storybook/manager-head.html:17]: Safari and Firefox don't support this way of defining font-face rules. I considered duplicating these definitions with `700` and `bold`, but it seems we don't have to. 
Good callout on the website, I'll fix that

Comment (spark33) [reply .storybook/manager-head.html:21]: Makes sense! Thanks for clarifying.

**Pull request merged**